client/src/app.jsx

```diff
@@ -25,7 +25,7 @@ class AddToCartApp extends React.Component {
       loading: true,
     }
     this.getGame = this.getGame.bind(this);
-    this.url = 'http://localhost:3001/api/'
+    this.url = 'http://localhost:3001/cartapi/'
```

Comment:
I think give the way we're setting up our proxying you want client GET requests to go to whatever is serving (either the proxy when it's being served that way, or your development index.html when you're serving there). So instead of having a full url here I think you just want a relative path.